app/common/translate.py

```diff
@@ -548,8 +615,7 @@ def clean_up_and_return_items(text: str) -> str:
     sanitized = re.sub("男は ", "", text)  # remove boy reference from start of string
     sanitized = re.sub("女は ", "", sanitized)  # remove girl reference from start of string
     sanitized = re.sub("男は　", "", sanitized)  # remove boy reference from start of string (fullwidth space)
-    sanitized = re.sub("女は　", "", sanitized)  # remove girl reference from start of string (fullwidth space)
-    sanitized = re.sub("！　と頼まれた。", "と頼まれた。", sanitized)  # remove annoying 'asked to' string
+    sanitized = re.sub("女は　", "", sanitized)  # remove girl reference from s tart of string (fullwidth space)
```

Comment:
Can you fix this comment? `s tart` -> `start

Reply:
Yep